waggle-dance-core/pom.xml

```diff
@@ -280,6 +281,21 @@
       <version>3.0.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-service</artifactId>
+      <version>3.1.1</version>
```

Comment:
Can this not use `${hive.version}` defined in the parent pom of this project? Otherwise we run the risk of having a mish-mash of Hive jar files with different versions.